apps/notification/models.py

```diff
@@ -73,3 +75,31 @@ class Meta:
     @staticmethod
     def get_for(user):
         return Notification.objects.filter(receiver=user).distinct()
+
+
+class Assignment(models.Model):
+    """
+    Assignment Model
+    """
+    timestamp = models.DateTimeField(
+        default=timezone.now,
+    )
```

Comment:
Let's use auto_now_add here.
https://docs.djangoproject.com/en/3.1/ref/models/fields/#django.db.models.DateField.auto_now_add

Also, can we change the timestamp to `created_at`? We are using that naming for most of the other models.